src/it/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCliComponentTest.java

```diff
@@ -65,7 +77,7 @@ public void checkCreateAndDiscoverComponent() throws IOException, InterruptedExc
         String application = APPLICATION_PREFIX + random.nextInt();
         String component = COMPONENT_PREFIX + random.nextInt();
         try {
-            createComponent(project, application, component, push);
+            createS2iComponent(project, application, component, push);
```

Comment:
test method name should be renamed as well

Reply:
i created a new set of parameters to test both S2i and Devfile components.